contents/handbook/growth/marketing/content.md

```diff
@@ -33,12 +33,12 @@ Content is the main pillar of our marketing strategy. Our strategy is to go _dee
 
 - The marketing team is responsible for all the above – mostly Andy, Ian, and Lior, but James Hawkins, Joe and Charles also contribute. The Website & Docs team owns our [reference documentation](/docs). 
 
-- You can view planned content on the [Marketing project board](https://github.com/orgs/PostHog/projects/8?card_filter_query=label%3Acontent) using the "content" label filter. All planned content should have an issue.
+- You can view planned content on the [Marketing project board](https://github.com/orgs/PostHog/projects/8?card_filter_query=label%3Acontent) (internally visible only) using the "content" label filter. All planned content should have an issue.
```

Comment:
Some content doesn't have the `content` tag FYI, so this link missed out on it. maybe just: https://github.com/orgs/PostHog/projects/8

Reply:
+1